engine/src/main/java/org/camunda/bpm/engine/impl/cfg/ConfigurationLogger.java

```diff
@@ -114,28 +115,26 @@ public void invalidPropertyValue(Exception e) {
   }
 
   /**
-   * Method for logging message when default history time to live is configured.
+   * Method for logging message when model TTL longer than global TTL.
    *
-   * @param definitionKey the correlated definition key with which history time to live is related to.
+   * @param definitionKey the correlated definition key with which history TTL is related to.
    *                      For processes related to httl, that is the processDefinitionKey, for cases the case definition key
    *                      whereas for decisions is the decision definition key.
    */
-  public void logHistoryTimeToLiveDefaultValueWarning(String definitionKey) {
-    String message = formatHTTLDefaultValueMessage(definitionKey);
-
-    logWarn("016", message);
+  public void logModelHTTLLongerThanGlobalConfiguration(String definitionKey) {
+    logWarn(
+        "017", "definitionKey: {}; "
```

Comment:
❓ Could you share why you've selected an incremented id from the previous log message (17 instead of 16)? Also does this have an impact?

Reply:
The new message is thrown under different prerequisites/circumstances. As we need to backport to 7.20, I decided to increase the warning number, just in case anyone relied on that. Maybe I am overthinking it.
An alternative approach would be to increase the number only for 7.20 but that will mean different numbers for the different versions.